### DISCUSSION TITLE
특정 아바타 크기 조절

### DISCUSSION THREAD

**Discussion by strangestraw** [naverz/zepeto-studio-kor#178] [5. Q&A]
안녕하세요.
제페토에서 특정 아바타(발표자, 공연자) 만 크기를 조절해서 보여줄 수 있을까요?
아래 영상과 같은 연출이 가능한 지 궁금합니다.
https://youtu.be/wYeFAlVC8qU
가이드 부탁드립니다.
감사합니다.

**Comment by x4cf** [comment]
시도해보지는 않았지만 불가능하지않은 기능으로보입니다

**Reply by strangestraw** [reply]
답변 감사드립니다!

**Comment by hyeonjin-jo** [comment]
안녕하세요.

특정 제페토 캐릭터의 크기를 조절하는 것은 scale값을 변경하면 가능합니다.

멀티플레이 기능이 들어가는 월드의 경우 스키마 타입에 scale값을 추가해서 연동하는 방식으로 진행 부탁드립니다.

**Reply by strangestraw** [reply]
답변 감사드립니다~
